model/postgres/postgres_server.rb

```diff
@@ -57,7 +57,11 @@ def configure_hash
     }
   end
 
+  def hostname(trailing_dot: false)
```

Comment:
Should we move this trailing_dot logic to the DnsZone? Probably all services will need it.